apps/web/src/components/Announcement/AnnouncementList.tsx

```diff
@@ -56,23 +56,25 @@ export default function AnnouncementList({ queryRef }: Props) {
       }
 
       if (!announcement.experienced) {
-        update({
-          type: announcement.key,
-          experienced: true,
-          optimisticExperiencesList: [
-            {
-              type: announcement.key,
-              experienced: true,
-            },
-          ],
-        });
+        dismissExperience(announcement.key);
       }
 
       hideDrawer();
     },
-    [hideDrawer, router, track, update]
+    [dismissExperience, hideDrawer, router, track]
   );
 
+  // TODO [GAL-3033]: come up with a system to "dismiss all announcement-type experiences"
```

Comment:
https://linear.app/galleryso/issue/GAL-3033/come-up-with-a-system-to-dismiss-all-announcement-type-experiences